### PR TITLE
Feature/iii 3465 Add similarityScore on events

### PR DESCRIPTION
**Pull request by geoffreydhuyvetters**
### Added

- Add similarity score on events create page

Ticket: https://jira.uitdatabank.be/browse/III-3465

### CODE REVIEW

File: functions/getMockedEventsSet.js [hunk 1]
```diff
@@ -407,7 +407,9 @@ const getMockedEventsSet = () => {
     suggestedEventsWithOneProduction,
   ];
   const index = Math.floor(Math.random() * events.length);
-  return events[index];
+  const similarity = Math.random();
```
Comment (simon-debruijn) [inline functions/getMockedEventsSet.js:410]: I know this is just dummy bet shouldn't it be a percentage like `Math.floor((Math.random() * 100) + 1)`

**Review by simon-debruijn — APPROVED** [review]

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -12,8 +12,6 @@
 
 ### Fixed
 
--
-
----
+- ***
```
Comment (bertramakers) [inline .github/pull_request_template.md:15]: I'm a bit confused, can you explain what this change is for? The way I see it you removed the horizontal line between the change log and the extra notes (ticket etc), and added `***` in the first bullet point of `Fixed`?

File: api/productions.js [hunk 3]
```diff
@@ -96,8 +96,7 @@ export const getSuggestedEvents = (
     return [];
```
Comment (bertramakers) [inline api/productions.js:96]: We should probably also update this to return an object now, right?

**Review by bertramakers — CHANGES_REQUESTED** [review]
I think the error handling in `getSuggestedEvents` will need to be changed now.

File: api/productions.js (hunk 3 above)
Comment (geoffreydhuyvetters) [reply api/productions.js:96]: True

File: .github/pull_request_template.md (hunk 2 above)
Comment (geoffreydhuyvetters) [reply .github/pull_request_template.md:15]: Hmm, this was done by prettier. Probably because it's an empty list

Comment (geoffreydhuyvetters) [reply .github/pull_request_template.md:15]: I'll revert it

**Review by bertramakers — APPROVED** [review]
Thanks for the quick changes @geoffreydhuyvetters, looks good to me now! ✅

**Pull request merged**